### PR TITLE
removes heart attacks from being too fat since it's not funny

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request
hearts will no longer fail from having the fat trait since it's mega uncontrollable, they still take damage from it though
people actually die to this like a lot

### Why is this change good for the game?
reduces RP
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
probably not documented but hearts will no longer fail solely due to being fat

### What general grouping does this PR fall under? 
medical

# Changelog

:cl:  
tweak: hearts will no longer spontaneously fail due to bad eating habits
/:cl:

### CODE REVIEW

**Comment by redmoogle** [comment]
not funny didnt laugh

**Comment by ToasterBiome** [comment]
unironically approve

**Comment by N3D6** [comment]
i can now inject 999 units of gravy into myself with no backfires

**Comment by redmoogle** [comment]
> 
> 
> i can now inject 999 units of gravy into myself with no backfires

dont make me make food have ODs

**Comment by N3D6** [comment]
> > i can now inject 999 units of gravy into myself with no backfires
> 
> dont make me make food have ODs

you wouldn't

**Comment by SomeguyManperson** [comment]
you can already do that safely if you also inject some vitamins

**Comment by Jarod1200** [comment]
Is this an ided pr

**Comment by maxion12345** [comment]
fat

**Comment by SomeguyManperson** [comment]
I wish this was an i ded pr but it's an "someone else ded pr"

**Comment by ChesterTheCheesy** [comment]
the heart attack is literally the only thing stopping you from eating indefinitely
if you eat too much, just go to fitness/holodeck and work that fat out

also, you literally removed heartattacks whatsoever, including random heart attacks and defibrillating a working heart

**Comment by ilikesaying** [comment]
wait, i could die as a podperson just by standing next to a light source for too long?

**Comment by SomeguyManperson** [comment]
no the nutrition from being in a lit area won't take you over the threshold to get fat

**Comment by JamieD1** [comment]
I think we should make better fat mechanics instead. Perhaps a treadmill so people can get rid of fatness? Considering its hard to know when you have eaten enough its a pain in the ass when you actually get fat.
